cli/dcoscli/marathon/main.py

```diff
@@ -99,6 +104,8 @@
     <task-id>                   The task id
 
     <scale-factor>              The factor to scale an application group by
+
+    <host>                      The host name to isolate your command to
```

Comment:
This should go in the `Options:` section as `--host=<host>`. It is not a positional argument.